README.md

```diff
@@ -1,8 +1,152 @@
+
+(venv) cabin$ bin/cabin status
+| version | table                             |   rows |    size | inputs               | outputs |
+|--------:|:----------------------------------|-------:|--------:|:---------------------|:--------|
+|    1  ! | StormDetailsTable::2011::ce78a5c3 | 79,101 | 19.4 MB | (1) StormDetailsFile | (0)     |
+|    2  ✓ | StormDetailsTable::2012::00ea4b32 | 64,508 | 15.2 MB | (1) StormDetailsFile | (0)     |
+```
+**aside** it is not necessary to manually update the version of the StormDetailsTable dataset because its import logic has not changed. We leave this to the user to decide as it is purely a communication tool to other developers to increase the version in downstream datasets or not. I have chosen to do so here because I find it communicates the idea more clearly for beginers - however leaving the version of StormDetailsTable as `1` would be perfectly reasonable and would all the same functionality.
+
+
+Say I actually wanted to change the *import logic* by adding new data columns, I would edit the code of `cabin/datasets/StormDetails.py` including changing the version from `2` to `3`. Note that:
+* the table `StormDetailsTable::2012::00ea4b32` is out of date
+* `import --all` does not download a new csv
+* for both datasets StormDetailsTable with root version 2012:
+    - status shows the same `rows` and `size`
+    - the formula sha part is not the same, there is a difference between datasets
+    - version 3 is up to date with the code, and hence is `latest ✓`
+
+```
+(venv) cabin$ bin/cabin status
+| version | table                             |   rows |    size | inputs               | outputs |
+|--------:|:----------------------------------|-------:|--------:|:---------------------|:--------|
+|    1  ! | StormDetailsTable::2011::ce78a5c3 | 79,101 | 19.4 MB | (1) StormDetailsFile | (0)     |
+|    2  ! | StormDetailsTable::2012::00ea4b32 | 64,508 | 15.2 MB | (1) StormDetailsFile | (0)     |
+
+(venv) cabin$ bin/cabin import --all
+[2022-01-13 14:12:41,510] INFO produce:  StormDetailsTable              with formula sha d6e14f82 and root versions 2012
+[2022-01-13 14:12:50,886] INFO Imported 64508 rows to table: StormDetailsTable::2012::d6e14f82 
+[2022-01-13 14:12:50,927] INFO produced: StormDetailsTable              with formula sha d6e14f82 and root versions 2012
+
+(venv) cabin$ bin/cabin status
+| version | table                             |   rows |    size | inputs               | outputs |
+|--------:|:----------------------------------|-------:|--------:|:---------------------|:--------|
+|    1  ! | StormDetailsTable::2011::ce78a5c3 | 79,101 | 19.4 MB | (1) StormDetailsFile | (0)     |
+|    2  ! | StormDetailsTable::2012::00ea4b32 | 64,508 | 15.2 MB | (1) StormDetailsFile | (0)     |
+|    3  ✓ | StormDetailsTable::2012::d6e14f82 | 64,508 | 15.2 MB | (1) StormDetailsFile | (0)     |
+```
```

Comment:
I don't think this code block is adding much value. It's mostly a repetition of what we have above. It's also become a state-dependent demo now; I need to have read up to this point to understand why `cabin status` is giving me two separate out of data imported tables.

Reply:
Sure I can take it out of the README. Maybe it can be added to a different doc later. 

The value: In the absence of any description or docs regarding changes to "import logic" versus "download logic" versus root version, I think examples are the best way to show what is expected behaviour.